### ISSUE TITLE
How to use the dashboard

### ISSUE THREAD

**Issue by popular1119** [mamroax/python_eit_w_gui#1]
Hello, I am very honored to see the article you shared, it is very well written. However, there is no usage guide in the article, it seems a bit confusing, can you post the usage guide? Sorry for the need, but it makes the reader understand what you mean. Looking forward to your reply!

I wish you all the best!

**Comment by mamroax** [comment]
Hello! Thank you for your message) I did not expect someone to use this
GUI(it is not completely mine tho, took it from customTkinter examples). I
will make a usage guide soon and update it w some new features

чт, 24 авг. 2023 г., 08:10 piao peng ***@***.***>:

**Comment by popular1119** [comment]
Hello, thank you very much for your contribution, and I am also looking forward to your upcoming user guide, which will be of great help to many people, and finally, thank you again for your reply!

**Comment by mamroax** [comment]
You're welcome! I added a short readme for the project and changed the
GUI to english. Yet it is not possible to choose your custom data file path
in GUI but it can be done inside a project. I will add some features in
future, so stay tuned. If you wish to share an opinion i set my contacts in
readme.

чт, 24 авг. 2023 г. в 12:31, piao peng ***@***.***>:

**Comment by popular1119** [comment]
I am very happy to hear from you, next, I will rerun the GUI according to the usage prompts, I believe it will have a good result, thank you for everything!

**Comment by popular1119** [comment]
Hi, with the help of the tutorial you uploaded yesterday, I have a deeper understanding of the role of this GUI, as you said: input a positive integer to average the measured value, the purpose is to remove noise. After I input a few integers in the GUI, I found that the "breath chart" has indeed changed, but the image in the visualization does not seem to change. Is this normal, or is the noise of the data itself not strong, so after the modification Nothing seems to have changed. Because the data collected by my device is not the same as the "experient.text" you placed, there is no way to evaluate it. If you have time, I look forward to your reply. Sorry for the inconvenience.
![image](https://github.com/mamroax/python_eit_w_gui/assets/126946185/5d2e1339-a38e-4474-85fc-ec22c16ab70c)

**Comment by mamroax** [comment]
![1](https://github.com/mamroax/python_eit_w_gui/assets/32238828/c7ef13f1-be18-4524-94e
![2](https://github.com/mamroax/python_eit_w_gui/assets/32238828/064cb7b5-55e9-4728-9dd7-46d43e33f1a8)
c-92fb8a7d243d)

As you already know, electrical impedance tomography is based on measuring the voltage difference across the electrodes. The resulting image of the impedance difference you see is based on the difference between two voltage points (high and low). The points correspond to inhalation and exhalation. This program currently takes the index of the frame with the lowest total frame voltage and the frame with the highest total voltage in a given file and compares them to each other. As you can see, with a sufficient voltage difference between the pictures, a difference is visible

**Comment by mamroax** [comment]
![1](https://github.com/mamroax/python_eit_w_gui/assets/32238828/1d95ee23-0f93-4c77-8fc4-b0525149ed5c)
The first image was not uploaded so here it is

**Comment by mamroax** [comment]
The number of frames was 1 and 50 to make the difference more noticeable. As you can see, 50 is not a good amount, because it "eats" all the valuable information about voltage peaks and drops. The difference would be even more noticeable if the recording took place in real time at 24 frames per second or more.

The idea was to smooth out small voltage fluctuations so that the graph and the resulting image of the conductivity difference  had no spikes and look better on real-time playback.

The difference between default 1 and 5 is not quite visible on a static image but it will be on a playback

**Comment by popular1119** [comment]
Thank you very much for your patience in answering, I have learned a lot. I don't know if you know an article published by MIT in 2021: "EIT-kit: An Electrical Impedance Tomography Toolkit for Health and Motion Sensing". I made a rough introduction to the equipment in this article, but now there are some problems. I don’t know if you have time and interest to discuss it. I look forward to your reply!
![Uploading image.png…]()
https://dl.acm.org/doi/abs/10.1145/3472749.3474758

**Comment by mamroax** [comment]
Yes, this is a very good article, I read it.  If you wish, we can connect on discord and discuss it

**Comment by popular1119** [comment]
Ok, here is my discord: pengpiao
![image](https://github.com/mamroax/python_eit_w_gui/assets/126946185/1cd6c368-b104-4a51-8baf-fe56812549fc)
